docs/getting-started-guides/scratch.md

```diff
@@ -0,0 +1,664 @@
+
+#### Selecting a Cluster Name
+
+You should pick a name for your cluster.  Pick a short name for each cluster
+which is unique from future cluster names. This will be used in several ways:
+  - by kubectl to distinguish between various clusters you have access to.  You will probably want a
+    second one sometime later, such as for testing new Kubernetes releases, running in a different
+region of the world, etc.
+  - Kubernetes clusters can create cloud provider resources (e.g. AWS ELBs) and different clusters
+    need to distinguish which resources each created.  Call this `CLUSTERNAME`.
+
+#### Selecting a Security Model
+
+There are two main options for security:
+ 1. access the apiserver using HTTP
+   - use a firewall for security
+   - easier to setup
+ 1. access the apiserver using HTTPS  
+   - use https with certs, and credentials for user
+   - recommended approach
+   - configuring certs can be tricky
+
+If following the HTTPS approach, you will need to prepare certs and credentials.
+
+##### Preparing Certs
+You need to prepare several certs:
+  - The master needs a cert to act as an HTTPS server.
+  - The kubelets optionally need certs to identify themselves as clients of the master, and when
+    serving its own API over HTTPS.
+
+Unless you plan to have a real CA generate your certs, you will need to generate a root cert and use that to sign the master, kubelet, and kubectl certs.
+    - see function `create-certs` in `cluster/gce/util.sh`
+    - see also `cluster/saltbase/salt/generate-cert/make-ca-cert.sh` and
+      `cluster/saltbase/salt/generate-cert/make-cert.sh`
+
+You will end up with the following files (we will use these variables later on)
+    - `CA_CERT`
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/ca.crt`.
+    - `MASTER_CERT`
+      - signed by CA_CERT
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/server.crt`
+    - MASTER_KEY 
+      - put in on node where apiserver runs, in e.g. `/srv/kubernetes/server.key`
+    - KUBELET_CERT
+      - optional
+    - KUBELET_KEY
+      - optional
+
+##### Preparing Credentials
+The admin user (and any users) need:
+  - a token or a password to identify them. 
+  - tokens are just long alphanumeric strings, e.g. 32 chars.  See
+    - `TOKEN=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)`
+
+Any tokens and.or passwords need to stored in a file for the apiserver
+to read.  This guide uses `/var/lib/kube-apiserver/known_tokens.csv`.
+The format for this file is described in the [authentication documentation](
+../authentication.md).
+
+For distributing credentials to clients, the convention in Kubernetes is to put the credentials
+into a [kubeconfig file](../kubeconfig-file.md).
+
+The kubeconfig file for the administrator can be created as follows:
+ - If you have already used Kubernetes with a non-custom cluster, you will already have a `$HOME/.kube/config` file.
```

Comment:
Insert after "non-custom cluster": "(for example, used a Getting Started Guide)"
... a non-custom cluster (for example, used a Getting Started Guide),...